templates/view_spec_helpers.rb

```diff
@@ -0,0 +1,21 @@
+module ViewSpecHelpers
+  def page
```

Comment:
@jferris @derekprior I opened a [PR on Capybara](https://github.com/jnicklas/capybara/pull/1533); please provide any feedback there, especially if you think this would be a helpful change.